spec/controller/api/before_actions_spec.rb

```diff
@@ -213,4 +223,22 @@ def index
       expect(run_action(base_klass, :index)).to match([200, instance_of(Hash), ["hi from base"]])
     end
   end
+
+  context 'case 7' do
+    let(:klass) { ControllerApiBeforeActionsSpec::TestController7 }
+
+    it 'executes the block as a method and adds it to before actions' do
+      expect { run_action(klass, :index) }.to output("Hello Rage").to_stdout
```

Comment:
That's actually interesting. I didn't know about the `output` helper.

However, do you want to update it to match the other tests? e.g. https://github.com/rage-rb/rage/blob/3a131858a2bf3ea02751d06189d40eb1b3e1150e/spec/controller/api/before_actions_spec.rb#L216-L219

Reply:
Yeahh, I just realized the print statement will just pollute the test screen. Will change to match other tests